Formula/go-critic.rb

```diff
@@ -21,6 +21,8 @@ class GoCritic < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "01aec52aee1170f6fbc2c304d2302884314de73df121c3d7199beb6d80da5aaa"
   end
 
+  disable! date: "2023-07-15", because: :does_not_build
+
```

Comment:
Only temporary, to be removed
```suggestion
```

Reply:
might be good to add a note in the code to reference the open issue, `https://github.com/go-critic/go-critic/issues/1349`